spec/factories/offsprings.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :offspring do
+    first_name "MyString"
+    last_name "MyString"
+    type ""
```

Comment:
The problem is that you are not following STI. You need to  create a new model that inherits from offspring. Delete the type as it is an error, it will be populated automatically